For_university/experiment1.py

```diff
@@ -0,0 +1,72 @@
+import random
+
+SIZE_OF_UNIVERSE = 100
+NUMBER_OF_SUBSETS = 40
+CARDINALITY = 4
+REMAINING_SUBSET = NUMBER_OF_SUBSETS - CARDINALITY
+ELEMENTS_OF_UNIVERSE = list(range(SIZE_OF_UNIVERSE))
+RATIO = int(NUMBER_OF_SUBSETS / CARDINALITY)
+NUMBER_OF_DUPLICATE = int((NUMBER_OF_SUBSETS-CARDINALITY)/(CARDINALITY/2))
+UNCOVERED_ELEMENTS = set(ELEMENTS_OF_UNIVERSE)
+TEST_LOOP = 1
+
+
+def making_universe(SIZE_OF_UNIVERSE, CARDINALITY):
+    universe = []
+    i = 0
+    small_subset = int(SIZE_OF_UNIVERSE / CARDINALITY - 1)
+    big_subset = int(SIZE_OF_UNIVERSE / CARDINALITY + 1)
+    last_element = 0
+    for index in range(CARDINALITY):
+        if index < int(CARDINALITY / 2):
+            subset = list(ELEMENTS_OF_UNIVERSE[last_element:last_element + big_subset])
+            for i in range(NUMBER_OF_DUPLICATE+1):
+                universe.append(subset)
+            last_element = last_element + big_subset
+        else:
+            universe.append(list(ELEMENTS_OF_UNIVERSE[last_element:last_element + small_subset]))
+            last_element = last_element + small_subset
+
+
+    return universe
+
+
+def coverage_function(subset):
```

Comment:
I think the word `function` is redundant. Ofcourse it's a function...
Work hard on succint, descriptive names!